### PR TITLE
Reformat paragraphs in the namespaces task page

### PR DESCRIPTION
**Pull request by tengqm**
This page can use more tweaks in addition to the ones proposed in this PR. This PR focuses on line wrappings, empty lines, inappropriate lists, and inappropriate indentations.

### CODE REVIEW

**Review by Shubham82 — CHANGES_REQUESTED** [review]

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 1]
```diff
@@ -131,191 +138,192 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
...
 
-
-
 <!-- discussion -->
 
 ## Understanding the motivation for using namespaces
 
-A single cluster should be able to satisfy the needs of multiple users or groups of users (henceforth a 'user community').
+A single cluster should be able to satisfy the needs of multiple users or groups of users
+(henceforth a 'user community').
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md:278]:
```suggestion
(henceforth in this document a _user community_).
```

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 2]
```diff
@@ -131,191 +138,192 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
...
 
 Kubernetes _namespaces_ help different projects, teams, or customers to share a Kubernetes cluster.
 
 It does this by providing the following:
 
 1. A scope for [Names](/docs/concepts/overview/working-with-objects/names/).
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md:284]:
```suggestion
1. A scope for [names](/docs/concepts/overview/working-with-objects/names/).
```

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Comment by shannonxtreme** [comment]
Is the `console` highlighter for output code blocks new? We don't seem to have a section in our style guide to advise new contributors about formatting code blocks...maybe worth an issue to track adding that. Otherwise, changes are good!

/lgtm

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**